model_utils/tracker.py

```diff
@@ -172,8 +175,16 @@ def changed(self):
             return {}
         saved = self.saved_data.items()
         current = self.current()
-        return dict((k, v) for k, v in saved if v != current[k])
-
+        dict = {}
```

Comment:
Let's not use `dict` as a variable name and shadow the built-in `dict` constructor. Something like `data` would be better.